tests/baselines/reference/emptyArrayBindingPatternParameter05.js

```diff
@@ -0,0 +1,26 @@
+//// [tests/cases/conformance/es6/destructuring/emptyArrayBindingPatternParameter05.ts] ////
+
+//// [emptyArrayBindingPatternParameter05.ts]
+export const cilBlurLinear: string[][] = [[]];
```

Comment:
It's a little hard to tell whats happening in the output of this test. Maybe introduce 3 different variables so it's clearer where and how they're being destructured.

Reply:
done